cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaMirrorMaker2Cluster.java

```diff
@@ -0,0 +1,316 @@
+                }
+            }
+        }
+
+        varList.add(buildEnvVar(ENV_VAR_KAFKA_MIRRORMAKER_2_CLUSTERS, clusterAliases.toString()));
+
+        if (clustersTrustedCerts.length() > 0) {
+            varList.add(buildEnvVar(ENV_VAR_KAFKA_MIRRORMAKER_2_TLS_CLUSTERS, "true"));
+            varList.add(buildEnvVar(ENV_VAR_KAFKA_MIRRORMAKER_2_TRUSTED_CERTS_CLUSTERS, clustersTrustedCerts.toString()));
+        }
+
+        if (clustersTlsAuthCerts.length() > 0 || clustersTlsAuthKeys.length() > 0) {
+            varList.add(buildEnvVar(ENV_VAR_KAFKA_MIRRORMAKER_2_TLS_AUTH_CLUSTERS, "true"));
+            varList.add(buildEnvVar(ENV_VAR_KAFKA_MIRRORMAKER_2_TLS_AUTH_CERTS_CLUSTERS, clustersTlsAuthCerts.toString()));
+            varList.add(buildEnvVar(ENV_VAR_KAFKA_MIRRORMAKER_2_TLS_AUTH_KEYS_CLUSTERS, clustersTlsAuthKeys.toString()));
+        }
+
+        if (clustersSaslPasswordFiles.length() > 0) {
+            varList.add(buildEnvVar(ENV_VAR_KAFKA_MIRRORMAKER_2_SASL_PASSWORD_FILES_CLUSTERS, clustersSaslPasswordFiles.toString()));
+        }
+
+        if (hasClusterOauthTrustedCerts) {
+            varList.add(buildEnvVar(ENV_VAR_KAFKA_MIRRORMAKER_2_OAUTH_TRUSTED_CERTS, "true"));
+        }
+
+        if (clustersOauthClientSecrets.length() > 0) {
+            varList.add(buildEnvVar(ENV_VAR_KAFKA_MIRRORMAKER_2_OAUTH_CLIENT_SECRETS_CLUSTERS, clustersOauthClientSecrets.toString()));
+        }
+
+        if (clustersOauthAccessTokens.length() > 0) {
+            varList.add(buildEnvVar(ENV_VAR_KAFKA_MIRRORMAKER_2_OAUTH_ACCESS_TOKENS_CLUSTERS, clustersOauthAccessTokens.toString()));
+        }
+
+        if (clustersOauthRefreshTokens.length() > 0) {
+            varList.add(buildEnvVar(ENV_VAR_KAFKA_MIRRORMAKER_2_OAUTH_REFRESH_TOKENS_CLUSTERS, clustersOauthRefreshTokens.toString()));
+        }
+
+        return varList;
+    }
+
+    private void getClusterTrustedCerts(final StringBuilder clustersTrustedCerts, KafkaMirrorMaker2ClusterSpec mirrorMaker2Cluster, String clusterAlias) {
+        KafkaMirrorMaker2Tls tls = mirrorMaker2Cluster.getTls();
+        if (tls != null) {
+            List<CertSecretSource> trustedCertificates = tls.getTrustedCertificates();
+   
+            if (trustedCertificates != null && trustedCertificates.size() > 0) {
+                if (clustersTrustedCerts.length() > 0) {
+                    clustersTrustedCerts.append("\n");
+                }
+                clustersTrustedCerts.append(clusterAlias);
+                clustersTrustedCerts.append("=");
+   
+                boolean separator = false;
+                for (CertSecretSource certSecretSource : trustedCertificates) {
+                    if (separator) {
+                        clustersTrustedCerts.append(";");
+                    }
+                    clustersTrustedCerts.append(certSecretSource.getSecretName() + "/" + certSecretSource.getCertificate());
```

Comment:
Use multiple append calls rather than mixing append and string concat with +

Reply:
Done